src/Util/ModuleUtils.php

```diff
@@ -65,6 +70,30 @@ public function getVersions(string $moduleName, bool $withAssetOnly = true): arr
         ), $versions);
     }
 
+    public function getFromBucket(): ModuleCollection
+    {
+        $modules = $this->bucket->objects(['prefix' => 'assets/modules/']);
+        $list = [];
+        foreach ($modules as $module) {
+            $parts = explode('/', $module->info()['name']);
+            if (!isset($list[$parts[count($parts) - 3]])) {
```

Comment:
Why 3 🤔

Reply:
I'll update this method so it will be clearer